README.md

```diff
@@ -95,13 +102,19 @@ Setting the environment variable LOG_LEVEL overrides the log level from the NODE
 
 ```typescript
 function getLevel(): Level | undefined {
-    switch (process.env.LOG_LEVEL?.toLowerCase()) {
-        case 'error': return Level.Error
-        case 'warn': return Level.Warn
-        case 'info': return Level.Info
-        case 'debug': return Level.Debug
-        case 'trace': return Level.Trace
-        default: return undefined
-    }
+  switch (process.env.LOG_LEVEL?.toLowerCase()) {
```

Comment:
I'm a bit confused as to why we need to show the implementation of this method in the README? 
The above text explains the functionality well. This just gives us more to maintain for no reason